articles/cloud-storage/cs-how-install-cifs-ecs.md

```diff
@@ -1,59 +1,59 @@
 ---
```

Comment:
Screen shots still say CIFS-ECS. Can those be changed now or is that something that will come later?